src/app/(sidebar)/account/create/page.tsx

```diff
@@ -46,27 +95,51 @@ export default function CreateAccount() {
             <Button size="md" variant="secondary" onClick={generateKeypair}>
```

Comment:
Sorry, one more thing here. When funding is in progress, let's disable the "Generate keypair" button. I like how you did that on the Fund page. :raised_hands: